huaweicloud/utils/validators.go

```diff
@@ -255,3 +255,21 @@ func ValidateECSTagValue(v interface{}, k string) (ws []string, errors []error)
 	}
 	return
 }
+
+func ValidateGaussDBVolumeSize(v interface{}, k string) (ws []string, errors []error) {
+	value := v.(int)
+	if value > 128000 {
```

Comment:
Does guessdb mysql support reducing capacity?

Reply:
not support.

Reply:
I took a look at the API, the maximum support is 1280000GB. It is a nice Validate.